src/components/sections/StandardSection.tsx

```diff
@@ -2,13 +2,14 @@ import { VerticalResultsDisplay } from '../VerticalResultsDisplay';
 import { SectionProps } from '../../models/sectionComponent';
 import { StandardCard } from '../cards/StandardCard';
 import { CompositionMethod, useComposedCssClasses } from '../../hooks/useComposedCssClasses';
+import { VerticalResultsCssClasses } from '../VerticalResults';
 
 /**
  * The CSS class interface used for {@link StandardSection}.
  *
  * @public
  */
-export interface StandardSectionCssClasses {
+export interface StandardSectionCssClasses extends VerticalResultsCssClasses {
   section?: string
 }
```

Comment:
Should we define `results___loading` in the StandardSection builtInCssClasses?

Reply:
50% opacity while loading universal results is already set in the `UniversalResults` component, which is why it doesn't need to be defined again with `results___loading` in the `StandardSection`.

I think having it at the higher-level component makes a little more sense, but, I don't have a strong preference. if it's confusing, I can remove the one from `UniversalResults` instead and define it in `StandardSection`

Reply:
Got it. In that case, should `results` or `results___loading` be in the VerticalResultsCssClasses then? I'm not seeing where they are being used

Reply:
Yes, both `results` and `results___loading` are part of `VerticalResultsCssClasses`.

Reply:
I see. I didn't notice that VerticalResults passes those classes to VerticalResultsDisplay through the `otherProps` variable